examples/echoxp.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+"""
+echoxp.py: test the echo application on two XIA hosts.
+One XIA host acts as the echo server and the other as echo client.
+The client sends a text or a file to the server which is echoed
+back to the client.
+"""
+
+import sys
+from mininet.log import setLogLevel, info
+from mininet.cli import CLI
+from mininet.link import Link
+from mininet.net import Mininet
+from mininet.node import XIAHost
+
+def topology():
+    "A topology for net-echo experiment"
+
+    net = Mininet( controller=None )
+
+    info( "*** creating nodes\n" )
+    xia1 = net.addHost( 'xia1', cls=XIAHost, hid='foo' )
```

Comment:
It would be nice to have a parameter `xdp=[]` in each of these calls to `addHost()`, just to load the XDP principal on each host. Then add the code to make XDPs loaded just like you did with HIDs below.